core/keys/symbols.py

```diff
@@ -0,0 +1,122 @@
+# define the spoken forms for symbols in command and dictation mode
+symbols_command_and_dictation = {}
+
+# for dragon, we add a couple of mappings that don't work for conformer
+# i.e. dragon supports some actual symbols as the spoken form
+dragon_symbols_command_and_dictation = {
+    "`": "`",
+    ",": ",",
+}
+
+# define the spoken forms for symbols that are intended for command mode only
+symbols_command = {}
+
+# define spoken form for symbols for use in create_spoken_forms.py functionality
+# we define a handful of symbol only. at present, this is restricted to one entry per symbol.
+symbols_for_create_spoken_forms = {
+    # for application names like "Movies & TV"
+    "and": "&",
+    # for emails
+    "at": "@",
+    "dot": ".",
+    # for application names like "notepad++"
+    "plus": "+",
+}
+
+
+class Symbol:
+    character: str
+    command_forms: list[str] = None
```

Comment:
Please have these in the same order as in the constructor. We read the order of the fields and were a bit confused about the definitions below.